meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -102,3 +102,6 @@ PREFERRED_VERSION_libstd-rs = "1.36.0"
 
 # balena-engine go version requirement
 GOVERSION = "1.12.17"
+
+# Set BALENA_STORAGE after config parsing for all recipes
+INHERIT += "balena-engine-storage"
```

Comment:
IMO this could be squashed with the previous commit but it's up to you.

Reply:
I prefer to separate introducing the class and using the class.